resources/knative-serving/values.yaml

```diff
@@ -10,8 +10,8 @@ global:
   containerRegistry:
     path: eu.gcr.io/kyma-project
   test_knative_serving_acceptance:
-    dir: develop/
-    version: 4a2b51ec
+    dir: pr/
+    version: PR-3657
```

Comment:
An image with a PR tag should not be merged to master.

Reply:
i know, i will update the PR as soon as i have that. I wanted to create a draft PR, but forgot during creation and GH does not allow you to do this afterwards.

but blocking the merge is a good idea, so i cannot forget it. THX 😄